src/input/Input.js

```diff
@@ -67,6 +71,12 @@ class Input extends Component {
 
     return (
       <View>
+        {
+          label != null &&
+          <Text style={[styles.label, labelStyle]}>
+            {label}
+          </Text>
```

Comment:
Should be the same as `label && element` not so? Any reason why explicitly checking for `null`?

Reply:
Yea I was thinking about the case where label is an empty string, but I can't think about any use case for that